### PR TITLE
 Remove DR jobs of AWS platform and add jobs for GCP

### PR DESCRIPTION
**Pull request by wangke19**
Remove DR jobs of AWS platform and add new jobs for GCP platform.

### CODE REVIEW

File: _releases/ocp-4.16-test-jobs-arm64.json [hunk 1]
```diff
@@ -1,9 +1,4 @@
 {
-  "nightly": [
```
Comment (rioliu-rh) [inline _releases/ocp-4.16-test-jobs-arm64.json:2]: @wangke19 suggest to submit a PR to disable prow job as well. if you won't add informing job for arm64

Comment (wangke19) [reply _releases/ocp-4.16-test-jobs-arm64.json:2]: For tight resources of x86_64, I choose to keep the job of arm64.

Comment (rioliu-rh) [reply _releases/ocp-4.16-test-jobs-arm64.json:2]: ok, so you will add another job in new PR for arm64?

Comment (wangke19) [reply _releases/ocp-4.16-test-jobs-arm64.json:2]: AWS DR jobs of amd64 is different from arm64, including FIPS. so it is better to disable prow job for arm64

**Comment by rioliu-rh** [comment]
/hold
@wangke19 will add new informing job for arm64

**Pull request merged**